src/junit/LIC_test.java

```diff
@@ -30,4 +31,36 @@ public void checkIfInvalidGivesFalse(){
         CMV cmv = new CMV(datapoints);
         assertFalse(cmv.lic5_calculate());
     }
+
+    @Test
+    public void test_lic9(){
```

Comment:
You should also test for when: 
* `C_PTS < 1`
* `D_PTS < 1`
* `C_PTS + D_PTS <= NUMPOINTS - 3`

Reply:
Additionally, explain in natural language what the test is doing. From [Canvas](https://canvas.kth.se/courses/37918/assignments/235345) it says:

* The contract that is tested is clearly formulated in natural language
* The assertions of the tests correctly and completely assess the contract

Reply:
Those are conditions on the parameters themselves, such parameters should never be passed to the function in the first place, I can't simply return false for those conditions as that is as arbitrary as returning true, it is just undefined

I can add a check that crashes the program if such parameters are passed (also the points_bound variable kind of ensures the same as the last condition ensures)

Regarding test comments, I'll add them